docs/guides/add-a-shared-dependency.md

```diff
@@ -0,0 +1,155 @@
+
+`@squide` aims to simplify the configuration of shared dependencies by abstracting the [fundamental shared dependencies](#default-shared-dependencies) necessary for building an application with React and React Router. Nevertheless, every federated application will inevitably require the configuration of additional custom shared dependencies.
+
+For a more comprehensive documentation of the Module Federation APIs, their functionality, and their benefits, please refer to this [article](https://www.infoxicator.com/en/module-federation-shared-api).
+
+## Default shared dependencies
+
+Since `@squide` has dependencies on React and React Router, the [define*](../reference/default.md#webpack) functions automatically configure shared dependencies for these packages by default, in addition to `@squide` own packages. The following shared dependencies are set as [singleton](#singleton-dependency) by default:
+
+- [react](https://www.npmjs.com/package/react)
+- [react-dom](https://www.npmjs.com/package/react-dom)
+- [react-router-dom](https://www.npmjs.com/package/react-router-dom)
+- [@squide/core](https://www.npmjs.com/package/@squide/core)
+- [@squide/react-router](https://www.npmjs.com/package/@squide/react-router)
+- [@squide/webpack-module-federation](https://www.npmjs.com/package/@squide/webpack-module-federation)
+
+For the full shared dependencies configuration, have a look at the [defineConfig.ts](https://github.com/gsoft-inc/wl-squide/blob/main/packages/webpack-module-federation/src/defineConfig.ts) file on Github.
+
+> You can [extend](../reference/webpack/defineDevHostConfig.md#extend-a-default-shared-dependency) or [override](../reference/webpack/defineDevHostConfig.md#override-a-default-shared-dependency) the default shared dependencies configuration.
+
+## What should be configured as a shared dependency?
+
+Candidates for shared dependencies:
+
+- Medium to large libraries that are used by multiple modules..
+- Libraries that requires a [single instance](#react--react-dom) to work properly (like `react`).
+- Libraries exporting [React contexts](#react-context-limitations).
+
+## Adding shared dependencies
+
+To configure shared dependencies, use the `sharedDependencies` option of any [define*](../reference/default.md#webpack) function:
+
+```js !#7-11 host/webpack.dev.js
+// @ts-check
+
+import { defineDevHostConfig } from "@squide/webpack-module-federation/defineConfig.js";
+import { swcConfig } from "./swc.dev.js";
+
+export default defineDevHostConfig(swcConfig, "host", 8080, {
+    sharedDependencies: {
+        "@sample/shared": {
+            singleton: true
+        }
+    }
+});
+```
+
+When a dependency is shared between a host application and a remote module, the sharing options must be **configured on both ends**. Let's also set the `@sample/shared` as a `singleton` for the remote module:
```

Comment:
> Let's also
The tone of this sentence feels different from the one before. 

The tone of the sentence before is neutral, `when X, Y must be set`
The tone of the sentence starting with lets seems more engage. `First we do this, then let's also set`

Reply:
It's because I wrote the first sentence during the day at 16h and wrote the second one at midnight 😆